build.sbt

```diff
@@ -400,5 +400,5 @@ lazy val publishSettings = Seq(
   )
 )
 
-addCommandAlias("fmt", "all scalafmtSbt scalafmt test:scalafmt")
-addCommandAlias("checkfmt", "all scalafmtSbtCheck scalafmtCheck test:scalafmtCheck")
+addCommandAlias("fmt", "all tofu/scalafmtSbt tofu/scalafmtAll")
```

Comment:
What's this about

Reply:
When you switch to some module and call `fmt` you may miss the files you touched outside this module (e.g. build.sbt or dependent modules) and then your PR fails.
The new setting is for GLOBAL formatting.